src/main/java/net/dv8tion/jda/api/requests/restaction/MessageAction.java

```diff
@@ -483,4 +516,143 @@ default MessageAction addFile(@Nonnull final File file, @Nonnull AttachmentOptio
     @Nonnull
     @CheckReturnValue
     MessageAction override(final boolean bool);
+
+    /**
+     * Sets the {@link net.dv8tion.jda.api.entities.Message.MentionType MentionTypes} that should be parsed.
+     * <br>If a message is sent with an empty Set of MentionTypes, then it will not ping any User, Role or {@code @everyone}/{@code @here},
+     * while still showing up as mention tag.
+     * <p>
+     * If {@code null} is provided to this method, then all Types will be pingable
+     * (unless whitelisting via one of the {@code mention*} methods is used).
+     * <p>
+     * Note: A default for this can be set using {@link #setDefaultMentions(Collection) MessageAction.setDefaultMentions(Collection)}.
+     *
+     * @param  allowedMentions
+     *         MentionTypes that are allowed to being parsed and pinged. {@code null} to disable and allow all mentions.
+     *
+     * @return Updated MessageAction for chaining convenience
+     */
+    @Nonnull
+    @CheckReturnValue
+    MessageAction allowedMentions(@Nullable Collection<Message.MentionType> allowedMentions);
+
+    /**
+     * Used to provide a whitelist for {@link net.dv8tion.jda.api.entities.User Users}, {@link net.dv8tion.jda.api.entities.Member Members}
+     * and {@link net.dv8tion.jda.api.entities.Role Roles} that should be pinged,
+     * even when they would not be pinged otherwise according to the Set of allowed mention types.
+     * <br>On other types of {@link net.dv8tion.jda.api.entities.IMentionable IMentionable}, this does nothing.
+     * <p>
+     * <b>Note:</b> When a User/Member is whitelisted this way, then parsing of User mentions is automatically disabled (same applies to Roles).
+     * <br>Also note that whitelisting users or roles implicitly disables parsing of other mentions, if not otherwise set via
```

Comment:
Does this mean doing `mention(user)` would disable mention parsing of roles? It doesn't seem to behave that way when looking through the code. Maybe this wording could be improved.

I do think it could make sense to disable all other mention types (unless specified otherwise by `allowedMentions`) when using this specific method, while using `mentionUsers(...)` would only disable parsing of user mentions. Thoughts?